samza-kafka/src/test/scala/org/apache/samza/system/kafka/TestKafkaSystemAdmin.scala

```diff
@@ -284,30 +353,6 @@ class TestKafkaSystemAdmin {
     assertEquals(3, partitionMetadata.replicas.size)
   }
 
-  class KafkaSystemAdminWithTopicMetadataError extends KafkaSystemAdmin(SYSTEM, brokerList, () => ZkUtils(zkConnect, 6000, 6000, zkSecure)) {
```

Comment:
It seems to me that the negative test cases are useful. Can we include similar negative test cases for the new KafkaSystemAdmin as well?

Reply:
Well, if we use new kafka consumer it should take care of all recoverable errors. Do we still need these tests?